src/lib/jsonFetcher.js

```diff
@@ -1,12 +1,17 @@
 var jQuery = (typeof $ !== 'undefined') ? $ : require('jquery') // funny little hack, sometimes it's in the global scope, sometimes it's maybe not...
-import extra_storage from './extra_storage'
-import defaultAdaptor from './AjaxAdaptorBase';
+import defaultAdaptor from './AjaxAdaptorBase'
 
-var _cleanSlashes = function(path) {
-  if (path.indexOf("/") === 0)
-    return path;
-  else
-    return "/" + path;
+function cleanSlashes(path) {
+  return `${path.indexOf('/') === 0 ? '' : '/'}${path}`
+}
+
+// TODO: Success data will only be a string if a test fixture returns it, and test side effects should not dictate the
```

Comment:
@BJK I just moved your comment up here as a more-noticeable TODO. I can move it back to where it was if you prefer.